test/integration/pkcs11-dbup.sh.nosetup

```diff
@@ -37,6 +37,11 @@ fi
 tar xzf "tpm2-pkcs11-$oldver.tar.gz"
 
 pushd "tpm2-pkcs11-$oldver"
+
+# Backport commit 260c84e3e9a1f44b09b690933e16900459739772
+# ("Drop unused variable which is causing the build to fail with current gcc10")
+sed -i 's/^} config;$/};/' src/lib/token.h
```

Comment:
Why don't I just do a minor release on 1.0.2 that contains this and then set oldver to 1.0.2. As long as we stay in the same minor version for oldver, the db schema is stable.

Reply:
Sure, that would obviously be the cleaner solution :+1:

Reply:
RC0 is rolled: https://github.com/tpm2-software/tpm2-pkcs11/releases/tag/1.0.2-RC0

Reply:
The release is done, but the image is updated to the newest IBM simulator which causes issues with the test suite yet again....